pkg/sign/timestamping.go

```diff
@@ -104,16 +107,21 @@ func (ta *TimestampAuthority) GetTimestamp(ctx context.Context, signature []byte
 		attempts++
 	}
 
+	body, err := io.ReadAll(response.Body)
```

Comment:
Nit but I would move the consumption of the body below where we have verified that we got an expected response (200 or 201).

Reply:
I'm undecided - if the error response contains a body with useful content, then we can return that in the error on line 115. I might leave this as-is.